saleor/graphql/product/mutations/products.py

```diff
@@ -522,3 +526,72 @@ class Meta:
     @classmethod
     def user_is_allowed(cls, user, input):
         return user.has_perm('product.manage_products')
+
+
+class VariantImageAssign(BaseMutation):
+    image = graphene.Field(
+        ProductImage, description='Product image beeing assigned.')
```

Comment:
'Assigned product image'? Anyway, `beeing` is either archaic, or maybe you meant
https://www.urbandictionary.com/define.php?term=beeing

Reply:
Obviously, it's a typo.